src/wtc-gallery-component.js

```diff
@@ -113,37 +112,40 @@ class Gallery extends ElementController {
       // otherwise, build a generic list of buttons
       if (this.options.paginationTarget) {
         itemList = document.querySelector(this.options.paginationTarget);
-        let items = itemList.children;
+        let items = [...itemList.children];
 
-        _u.forEachNode(items, (index, el) => {
+        items.forEach((el, index) => {
```

Comment:
Is there a good reason to use `index` instead of `i`? Generally `i` is well understood to be index/iteration.

Reply:
I'm happy to change it to `i`. A lot of the exitsting code uses `index`, but I agree—we don't need to do that. Will change

Reply:
Changed this

Reply:
Sounds good. This was more about inconsistency throughout the library than anything else. Thanks.